nx584_client

```diff
@@ -43,14 +43,14 @@ def do_arm(clnt, args):
         mode = 'exit'
     else:
         mode = 'auto'
-    clnt.arm(mode)
+    clnt.arm(mode, int(args.partition))
```

Comment:
For the client side, this should be the only place you need the type conversion.

Reply:
Actually, I'm wrong because argparse handles it, but.. not a huge deal to leave this.